### DISCUSSION TITLE
Nested transaction inside isolate

### DISCUSSION THREAD

**Discussion by AnisovAleksey** [simolus3/drift#2414] [Q&A]
Hello
Is there any way to run nested transaction inside [computeWithDatabase](https://drift.simonbinder.eu/api/isolate/computewithdriftisolate/computewithdatabase) ? if transaction started outside

**Comment by AnisovAleksey** [comment]
wow, I found this [commit](https://github.com/simolus3/drift/commit/3ee1c1ccd4465e44165c787ca6247d88d21ee7c6) and it's worked with these changes, thank you 👍 🥇

**Reply by simolus3** [reply]
Released as part of `drift: 2.8.0` (if you've been using a dependency override to git).
